ihtcvirtualgipssolution/src/ihtcvirtualgipssolution/Model.gipsl

```diff
@@ -96,7 +96,11 @@ rule selectOperationDay {
 		.isSelected := true
 	}
 	
-	c : Capacity 
+	c : Capacity {
```

Comment:
This is more of a general question: did you need to add the changes in some patterns for the translation of `filter` to `join`?

If this is not the case, we should leave the patterns as they are and only adapt the expressions in GIPSL. (Then they are "more" consistent with what we submitted to the journal.)